lib/PuppeteerSharp/BrowserContext.cs

```diff
@@ -83,6 +107,46 @@ public Task CloseAsync()
             return Browser.DisposeContextAsync(_id);
         }
 
+        /// <summary>
+        /// Overrides the browser context permissions.
+        /// </summary>
+        /// <returns>The task.</returns>
+        /// <param name="origin">The origin to grant permissions to, e.g. "https://example.com"</param>
+        /// <param name="permissions">
+        /// An array of permissions to grant. All permissions that are not listed here will be automatically denied.
+        /// </param>
+        /// <example>
+        /// <![CDATA[
+        /// var context = browser.DefaultBrowserContext;
+        /// await context.OverridePermissionsAsync("https://html5demos.com", new List<string> {"geolocation"});
+        /// ]]>
+        /// </example>
+        /// <seealso href="https://developer.mozilla.org/en-US/docs/Glossary/Origin"/>
+        public async Task OverridePermissionsAsync(string origin, IEnumerable<OverridePermission> permissions)
+        {
+            var permissionList = permissions.Select(permission =>
```

Comment:
since `permissions` is strongly typed could we just add an `EnumMember` attribute to each enum member?
if this works maybe also cleanup [`WaitUntilNavigation`](https://github.com/kblok/puppeteer-sharp/blob/master/lib/PuppeteerSharp/NavigatorWatcher.cs#L11) ?